var/spack/repos/builtin/packages/py-uproot3-methods/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyUproot3Methods(PythonPackage):
```

Comment:
Not needed if you use `py-uproot` instead of `py-uproot3`

Reply:
It is needed, as py-uproot3 is needed.